api/src/main/java/org/motechproject/nms/api/web/contract/mobileAcademy/CourseVersionResponse.java

```diff
@@ -0,0 +1,28 @@
+package org.motechproject.nms.api.web.contract.mobileAcademy;
+
+/**
+ * API response object for getting course version
+ */
+public class CourseVersionResponse {
+
+    private int courseVersion;
+
+    public CourseVersionResponse(){
+    }
+
+    public CourseVersionResponse(int courseVersion) {
+        this.courseVersion = courseVersion;
+    }
+
+    public CourseVersionResponse(Integer courseVersion) {
+        this.courseVersion = courseVersion;
+    }
+
```

Comment:
No, oversight on my part. I was playing around with return types before our team decided on non-nullable returns for api response for required fields. Will fix.